docs/use-cases/defi/integrate-with-xrpl-dex.md

```diff
@@ -0,0 +1,105 @@
+---
+html: integrate-with-xrpl-dex.html
+parent: defi-uc.html
+blurb: Integrate with the XRPL DEX.
+labels:
+  - Decentralized Exchange
+---
+# Integrate with the XRPL Decentralized Exchange
+
+Integrating with the XRP Ledger gives you access to one of the world's oldest decentralized exchanges (DEX), operating continuously since 2012. Users can trade [tokens](tokens.html) and XRP with minimal [fees](fees.html) to the network itself.
+
+To learn more about the XRPL DEX structure and limitations, see: [Decentralized Exchange](decentralized-exchange.html).
+
+
+## Set Up Node Services
+
+For lighter use cases, you can often rely on free public servers. However, more serious use of the XRP Ledger requires your own infrastructure.
```

Comment:
Suggest rewording "lighter use cases" and "more serious use" to something like "less complex applications/simpler scenarios" and "more substantial applications/critical use cases"